src/controllers/IllnessStartDateController.ts

```diff
@@ -0,0 +1,64 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+
+import { BaseController } from 'app/controllers/BaseController';
+import { StartDateValidator } from 'app/controllers/validators/StartDateValidator';
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { Reasons } from 'app/models/Reasons';
+import { Feature } from 'app/utils/Feature';
+import { ILLNESS_START_DATE_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template: string = 'illness/illness-start-date';
+
+const navigation: Navigation = {
+    previous(): string {
+        return ILLNESS_START_DATE_PAGE_URI;
+    },
+    next(): string {
+        return ILLNESS_START_DATE_PAGE_URI;
+    }
+};
+
+const applyPadding = (dayMonth: string): string => {
+    return dayMonth.length < 2 ? dayMonth.padStart(2, '0') : dayMonth;
+};
+
+@controller(ILLNESS_START_DATE_PAGE_URI, SessionMiddleware, AuthMiddleware,
+    FeatureToggleMiddleware(Feature.ILLNESS_REASON))
+export class IllnessStartDateController extends BaseController<Illness> {
```

Comment:
Using `Illness` interface here is not quite correct cause `Illness` class has many more properties than what is available via `request.body` (form data structure).

In other PR it was implemented as:

<img width="746" alt="image" src="https://user-images.githubusercontent.com/1724691/88918120-dc200200-d260-11ea-913d-ebbebb1fcc04.png">

In that instance internal interface describes data structure that HTML form created.